packages/suite/src/dsl.ts

```diff
@@ -11,8 +11,10 @@ export function test<C extends Context>(description: string): TestBuilder<C> {
 
 export class TestBuilder<C extends Context> implements TestImplementation {
   public description: string;
-  public steps: Step[];
-  public assertions: Assertion[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public steps: Step<any, any>[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public assertions: Assertion<any>[];
```

Comment:
I couldn't figure out how to get the collection of steps to type correctly without putting an any in here 🤷

Reply:
As mentioned [here](https://github.com/thefrontside/bigtest/pull/504/files#r484054190) if we don't actually modify these types, we don't need to invent type parameters here.